img/fieldScanInfo

```diff
@@ -0,0 +1 @@
+fieldScanInfo.dds
```

Comment:
I merged both your pull requests and started a new game and FS crashed. I believe it is the fieldScanInfo.dds.